docs/concepts/artifact.mdx

```diff
@@ -0,0 +1,239 @@
+}
+```
+
+### Annotations only artifacts
+
+Artifacts may store metadata in the manifest as annotations and do not have a config or blobs.
+For these the `artifactType` property is used to declare the type of the artifact.
+The config property is required in the manifest for maximum compatiblity an empty layer is also created.
+as per the  [empty descriptors guidance][empty-descriptor].  
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.example+type",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/vnd.oci.empty.v1+json",
+      "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+      "size": 2
+    }
+  ],
+  "annotations": {
+    "oci.opencontainers.image.created": "2023-01-02T03:04:05Z",
+    "com.example.info": "useful-info"
+  }
+}
+```
+
+### Harnessing Image Indexes
+
+One of the key advantages of OCI artifacts is their ability to utilize [Image Indexes][image-index],
+allowing the bundling of various OCI artifacts. An example use case if is when you want to represent
+multiple SBOMs of different architectures under one unified artifact.
+
+The Image Index is a higher-level construct in the OCI Image Specification that can point to
+multiple Image Manifests, each suitable for a different platform or architecture. It helps to
+group related artifacts together, providing a single entry-point for accessing any specific
+artifact depending on the required architecture.
+
+Below is an example of an OCI Image Index for an artifact that represents multiple SBOMs
+of different architectures:
+
+```json 
+{
+  "schemaVersion": 2,
+  "artifactType": "application/vnd.example.sbom.v1",
+  "manifests": [
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "artifactType": "application/vnd.example.sbom.v1",
+      "size": 7143,
+      "digest": "sha256:1111...",
+      "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+      },
+    },
+    {
+      "mediaType": "application/vnd.oci.image.manifest.v1+json",
+      "artifactType": "application/vnd.example.sbom.v1",
+      "size": 7682,
+      "digest": "sha256:2222...",
+      "platform": {
+        "architecture": "arm64",
+        "os": "linux"
+      },
+    }
+  ]
+}
+```
+
+## Best Practices and Limitations
+
+While working with OCI artifacts, keep in mind that not all registries have implemented
+image-spec v1.1. For backward compatibility, OCI artifacts can use the `config.mediaType`
+to represent the artifact type, aligning with the IANA mediaType of the artifact.
+
+In cases where the creation of an artifact doesn't involve a config blob, make sure to use
+the `artifactType` field in the manifest. If a config blob is indispensable, comply with
```

Comment:
I see you're trying to explain the balance of `artifactType` and `config.mediaType`. The paragraph as worded is a bit confusing. 

If we're worried about registries, including Docker Hub, that doesn't _currently_ support the `artifactType` property, the `config.mediaType` is the valid fallback. Which has no scenario loss. 
The wording around comply with the image specification is where I got lost.

The next line for keeping the manifest size manageable is another great point.  Could these be bulleted lists?

- Not all registries have implemented support for the `artifactType` property. For maximum portability, use the `config.mediaType` property to declare the Artifact Type
- For registries that do support the `artifactType`, the config.mediaType can be set to `"application/vnd.oci.empty.v1+json"`
- As a rule of thumb, keep the manifest size manageable by avoiding excessive annotations and leveraging blobs for larger data chunks.